helm/values.yaml

```diff
@@ -108,6 +108,11 @@ namespacedValidation:
   enabled: false
   mode: ignore  # 'ignore' or 'validate'
 
+# PodSecurityPolicy for Kubernetes cluster using this feature
+psp:
```

Comment:
I think the name should be explicit and self-explanatory as `podSecurityPolicy`.
Furthermore, I would consider PodSecurityPolicy a part of the `deployment` section at the beginning of `helm/values.yaml`, as it only really affects the actual deployment of Connaisseur itself and does not hold any value for its features. I would propose to append to the end of `deployment` as `deployment.podSecurityPolicy`.
Note that the templates above will require adjustment.